src/adapters/TelnetAdapter.ts

```diff
@@ -783,17 +803,6 @@ export class TelnetAdapter {
                 //we have reached the end of the collection. scrap children because they need evaluated in a separate call to compute their types
                 child.children = [];
 
-                if (isRoSGNode) {
-                    let nodeChildrenProperty = <EvaluateContainer>{
```

Comment:
This is the primary fix for this PR. We were adding this too early, which was preventing us from running the command for the rest of the properties at a later time.